deploy/bootstrap/users.sh

```diff
@@ -0,0 +1,59 @@
+#!/usr/bin/env bash
+
+
+( # run in subshell
+    set -e
+
+    pushd ${HERE}/../../tests/_common &>/dev/null
+
+    USERS_DIR=$(pwd)/users # includes expansion
+
+    rm -rf ${USERS_DIR}
+    mkdir -p ${USERS_DIR}
+
+    declare -A USERS=([dummy]=15001 [john]=15002 [jane]=15003)
+    declare -A PASSPHRASES
+
+    # check that bcrypt is installed for this python version
+    if ! ${PYTHONEXEC} -c 'from importlib.util import find_spec; import sys; sys.exit(0 if find_spec("bcrypt") is not None else 1)'; then
```

Comment:
This should be able do be done in a prettier way than a combined bash python oneliner.